DECISIONFILE_ENTRY.md

```diff
@@ -0,0 +1,13 @@
+template format for decision files
+
+DATE in YYYY MMM DD
+
+MOTION (mandatory field)
+
+PROPOSER
+
+SUMMARY/DISCUSSION (link to email thread if applicable)
+
+STATUS
```

Comment:
For consistency will there be a limited number of options here- eg approved/in progress/rejected/etc?